MCHI/JORPanel.cs

```diff
@@ -321,7 +325,7 @@ private Control MakeControl(JORControl jorControl)
             else if (jorControl.Type == "GRBX")
             {
                 var g = new GroupBox();
-                g.Text = jorControl.Name;
+                g.Text = transDict.Translate(jorControl.Name, jorControl); // i guess groupboxes can have text? uh guess we'll translate
```

Comment:
fyi this never happens in Twilight Princess, they only use GRBX as separators, but the function to generate them does take a char* for some reason